threadpool/core/src/main/java/cn/hippo4j/core/enable/BeforeCheckConfiguration.java

```diff
@@ -39,6 +40,26 @@ public class BeforeCheckConfiguration {
     private final String bootstrapPropertiesClassName = "cn.hippo4j.springboot.starter.config.BootstrapProperties";
     private final String bootstrapConfigPropertiesClassName = "cn.hippo4j.threadpool.dynamic.mode.config.properties.BootstrapConfigProperties";
 
+    private static final String POLARIS_NAMESPACE = "${spring.dynamic.thread-pool.polaris.namespace}";
```

Comment:
The same configuration and constants exist in `PolarisRefresherHandler`, so maybe it's easier to maintain the code without having to maintain two copies of the same configuration.

Reply:
Thanks for your guidance, but I have a question that when the configuration file contains the field and the value is the default value, how to distinguish whether the default value is used?

Reply:
So I think if you add a preemptive null judgement isn't it possible to remove the default values

Reply:
So I think if you add a preemptive null judgement isn't it possible to remove the default values